Remove yellow pages support

This support requires yellow pages support in libtirpc, which isn't
available. As most Buildroot users are most likely never going to need
such feature, get rid of it.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Index: b/src/security.c
===================================================================
--- a/src/security.c
+++ b/src/security.c
@@ -23,9 +23,6 @@
 #include <rpcsvc/mount.h>
 #include <rpcsvc/rquota.h>
 #include <rpcsvc/nfs_prot.h>
-#include <rpcsvc/yp.h>
-#include <rpcsvc/ypclnt.h>
-#include <rpcsvc/yppasswd.h>
 
 #include "rpcbind.h"
 
@@ -322,24 +319,9 @@
 		    args->rmt_proc != MOUNTPROC_UMNT)
 			break;
 		goto deny;
-	case YPBINDPROG:
-		if (args->rmt_proc != YPBINDPROC_SETDOM)
-			break;
-		/* FALLTHROUGH */
-	case YPPASSWDPROG:
 	case NFS_PROGRAM:
 	case RQUOTAPROG:
 		goto deny;
-	case YPPROG:
-		switch (args->rmt_proc) {
-		case YPPROC_ALL:
-		case YPPROC_MATCH:
-		case YPPROC_FIRST:
-		case YPPROC_NEXT:
-			goto deny;
-		default:
-			break;
-		}
 	default:
 		break;
 	}
